### PR TITLE
feat(judicial-system): Investigation Notifications

### PR DESCRIPTION
**Pull request by gudjong**
# Notifications for Investigation Cases

https://app.asana.com/0/1199153462262248/1200403423569995

## What

☝️ 

## Why

Part of the requirements for investigation cases

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/web/src/utils/hooks/use-case/useCase.tsx [hunk 1]
```diff
@@ -30,33 +31,47 @@ type autofillProperties = Pick<
...
 
-interface CreateCourtCaseMutationResponse {
-  createCourtCase: {
-    courtCaseNumber: string
-  }
+interface CreateCaseMutationResponse {
+  createCase: Case
+}
+
+interface CreateCourtCaseMutationReqponse {
```
Comment (oddsson) [inline apps/judicial-system/web/src/utils/hooks/use-case/useCase.tsx:38]: Typo in interface name

File: apps/judicial-system/web/src/utils/hooks/use-case/useCase.tsx [hunk 2]
```diff
@@ -30,33 +31,47 @@ type autofillProperties = Pick<
...
-  ] = useMutation(TransitionCaseMutation)
+    createCaseMutation,
+    { loading: isCreatingCase },
+  ] = useMutation<CreateCaseMutationResponse>(CreateCaseMutation)
   const [
     createCourtCaseMutation,
-    { loading: creatingCourtCase },
-  ] = useMutation<CreateCourtCaseMutationResponse>(CreateCourtCaseMutation)
+    { loading: isCreatingCourtCase },
+  ] = useMutation<CreateCourtCaseMutationReqponse>(CreateCourtCaseMutation)
```
Comment (oddsson) [inline apps/judicial-system/web/src/utils/hooks/use-case/useCase.tsx:62]: Same

**Review by oddsson — CHANGES_REQUESTED** [review]
Small typo but otherwise, looks awesome!

File: apps/judicial-system/web/src/utils/hooks/use-case/useCase.tsx (hunk 1 above)
Comment (gudjong) [reply apps/judicial-system/web/src/utils/hooks/use-case/useCase.tsx]: fixed

File: apps/judicial-system/web/src/utils/hooks/use-case/useCase.tsx (hunk 2 above)
Comment (gudjong) [reply apps/judicial-system/web/src/utils/hooks/use-case/useCase.tsx]: fixed

**Review by oddsson — APPROVED** [review]

**Pull request merged**